external-crates/move/crates/move-compiler/tests/move_2024/expansion/self_leading_access.move

```diff
@@ -0,0 +1,8 @@
+// module member aliases do not shadow leading access names
+module a::foo {
+    public struct S()
+    public fun foo(): S {
+        // TOOD fix this, should resolve to a::m::S
```

Comment:
This shouldn't be too bad: a module just needs to add itself as a leading access name to the alias map builder, right?

Reply:
Yeah I'll take care of this in a followup PR, just didn't want do too much for now